receiver/webhookeventreceiver/req_to_log.go

```diff
@@ -29,6 +31,7 @@ func reqToLog(sc *bufio.Scanner,
 
 	for sc.Scan() {
 		logRecord := scopeLog.LogRecords().AppendEmpty()
+		logRecord.SetTimestamp(pcommon.NewTimestampFromTime(time.Now()))
```

Comment:
I think you should rather set the [Observed Timestamp](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.34.0/specification/logs/data-model.md#field-observedtimestamp) field and not the [Timestamp](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.34.0/specification/logs/data-model.md#field-timestamp) field.

Reply:
I struggled a little with which was the more accurate field -- from the frame of reference of the collector the event is "occuring" when the event enters the collector (Timestamp) because the log is created from the event. However from a user's perspective Observed Timestamp is probably a more intuitively defined field.

Reply:
I'd say ObservedTimestamp is more appropriate. Love to see this merged!

Reply:
Its been fixed! Thanks for your patience sorry for the delay!